src/shapes/ActiveSelection.ts

```diff
@@ -43,11 +43,15 @@ export class ActiveSelection extends Group {
     //  noop
   }
 
+  selectAll(...targets: FabricObject[]) {
+    this.add(...targets);
+  }
```

Comment:
if we have multiSelect already i don't see to make sense.
This is a selectAll that takes targets as an input? so what the ALL part does? all of what?

Reply:
All of the objects in the selection...
Naming can be modified.
Also we can use multiSelect and pass more context so that the method knows what calle it

Reply:
It's weird to me that a method called `selectAll` doesn't replace the existing selection, instead it adds to the selection. Should be called `addAll` then? Also not clear what's its usage exactly if one doesn't need to override it.